crates/hulk_nao/src/main.rs

```diff
@@ -39,12 +45,18 @@ pub fn setup_logger() -> Result<(), fern::InitError> {
     Ok(())
 }
 
+#[derive(Parser)]
+struct Arguments {
+    #[arg(short, long, default_value = "logs")]
+    log_path: String,
+
+    #[arg(short, long, default_value = "etc/parameters/framework.json")]
+    framework_parameters_path: String,
+}
```

Comment:
Does this work?

```suggestion
struct Arguments {
    #[arg(short, long, default_value = "logs")]
    log_path: PathBuf,

    #[arg(short, long, default_value = "etc/parameters/framework.json")]
    framework_parameters_path: PathBuf,
}
```

Reply:
I tried that first but sadly not, the field type must support Display, which PathBuf doesn't.

Reply:
Huh...
Looks like this doesn't work with `default_value_t`, but does with `default_value`, changed.